setup/Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles/CommonFiles.swr

```diff
@@ -657,3 +657,6 @@ folder "InstallDir:MSBuild\Microsoft\VisualStudio\Managed\zh-Hant"
   file source="$(VisualStudioXamlRulesDir)zh-Hant\ResolvedProjectReference.xaml"
   file source="$(VisualStudioXamlRulesDir)zh-Hant\ResolvedSdkReference.xaml"
   file source="$(VisualStudioXamlRulesDir)zh-Hant\SdkReference.xaml"
+
+  folder "Extensions\Microsoft\ManagedProjectSystem\UnifiedSettings"
```

Comment:
We're required to place the settings in this specific place: ExtensionRoot\UnifiedSettings\*.registration.json